pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -1008,16 +1053,39 @@ func (m *kubeGenericRuntimeManager) GetPodStatus(uid kubetypes.UID, name, namesp
 		if idx == 0 && resp.Status.State == runtimeapi.PodSandboxState_SANDBOX_READY {
 			podIPs = m.determinePodSandboxIPs(namespace, name, resp.Status)
 		}
+
+		if idx == 0 && utilfeature.DefaultFeatureGate.Enabled(features.EventedPLEG) {
+			if resp.Timestamp == 0 {
+				containerStatuses, err = m.getPodContainerStatuses(uid, name, namespace)
```

Comment:
making sure I understand this correctly: in most cases this shouldn't happen, in other words, timestamp should always be set when container runtime responds. This is needed in case:
a. an old version of container runtime is used, or 
b. a bug somewhere caused missing timestamp.

Is my understanding correct?

Reply:
Yes, that's correct.

Reply:
Can we add a comment for this?

Reply:
I am not sure if we need this. If we want to keep this, let's log as this isn't expected to happen if things are working correclty.

Reply:
@mrunalp When the `Evented PLEG` is enabled in the kubelet, but if someone uses an older runtime which doesn't implement the latest changes from CRI API (which are part of this PR), then the timestamp value will not be set. 

A good example of this is the CI job `pull-kubernetes-e2e-gce-alpha-features`. In this job they would enable all alpha feature gates, which would include `Evented PLEG` as well, but they will still run it with the older containerd runtime which does not have `Evented PLEG` or the CRI API changes from this PR. Without this change that job would fail. 

I have added an additional log and a comment in the code to describe this scenario better. Thanks.